test/aircraft/FlightManagementSystem/RouteModel.spec.js

```diff
@@ -1020,6 +1040,16 @@ ava('._createAmendedDivergentLeg() returns an empty array when leg is a direct l
     t.deepEqual(result, []);
 });
 
+ava('._createAmendedDivergentLeg() returns the leg unmodified when divergent waypoint is the SID leg\'s last fix', (t) => {
+    const model = new RouteModel('KLAS07R.BOACH6.HEC');
+    const waypointName = 'HEC';
+    const legIndex = 0;
+    const expectedResult = [model._legCollection[legIndex]];
```

Comment:
You should specify the expected result inline here instead of relying on the model to produce it for you.

Reply:
done